tests/test_002-exclusive_jets.py

```diff
@@ -4,6 +4,8 @@
 
 import fastjet._pyjet  # noqa: F401
 
+dak = pytest.importorskip("dask_awkward")  # noqa: F401
+distributed = pytest.importorskip("distributed")
```

Comment:
ah! don't do this here - it'll skip the whole test.py file if you are missing dask_awkward / distributed /etc.

Stick this test in the dask tests!

Reply:
Ah sorry about that! I've moved the test to test_008, and reverted test_002